src/Runner.Worker/ContainerOperationProvider.cs

```diff
@@ -192,13 +192,12 @@ private async Task StartContainerAsync(IExecutionContext executionContext, Conta
             {
                 Trace.Info($"User provided port: {port.Value}");
             }
-            foreach (var volume in container.UserMountVolumes)
+            foreach (var mount in container.UserMountVolumes)
             {
-                Trace.Info($"User provided volume: {volume.Value}");
-                var mount = new MountVolume(volume.Value);
+                Trace.Info($"User provided volume: {mount.UserProvidedValue}");
```

Comment:
I tested we still print the same `[2022-05-13 13:00:38Z INFO ContainerOperationProvider] User provided volume: /home/test:/home/test/` message